internal/cmd/organizations/list.go

```diff
@@ -35,36 +34,8 @@ func listOrgsCommand() *cobra.Command {
 				return fmt.Errorf("failed to list organizations: %w", err)
 			}
 
-			// TODO: We should look at abstracting the formatting here into a library
-			//       that can be used by multiple commands needing to offer multiple
-			//       output formats from a command.
-			switch outputFormat {
-			case "yaml":
-				marshaller := &protoyaml.MarshalOptions{
-					Indent: 2,
-				}
-				output, err := marshaller.Marshal(listOrgs)
-				if err != nil {
-					return fmt.Errorf("failed to list organizations: %w", err)
-				}
-				fmt.Print(string(output))
-			case "json":
-				output, err := protojson.Marshal(listOrgs)
-				if err != nil {
-					return fmt.Errorf("failed to list organizations: %w", err)
-				}
-				fmt.Print(string(output))
-			case "table":
-				orgTable := table.New("DISPLAY NAME", "RESOURCE ID")
-				if len(listOrgs.Organizations) == 0 {
-					fmt.Printf("No organizations found")
-				} else {
-					for _, org := range listOrgs.Organizations {
-						orgTable.AddRow(org.DisplayName, org.OrganizationId)
-					}
-				}
-				orgTable.Print()
-			}
+			outTableHeaders, outTableData := getListOrganizationsTableOutputData(listOrgs)
+			output.CLIPrint(os.Stdout, outputFormat, listOrgs, outTableHeaders, outTableData)
```

Comment:
Not sure if we want os.Stderr here. I only added this input so we can more easily test the output package.

Reply:
Alternatively, I can create a struct/interface for the output package that has a WithWriter() method so we can override it only when we want to. I just thought to minimize struct initializers for simpler code.

Reply:
I don't have a strong preference right now, but think passing in an `io.Writer` is a fine approach (pretty similar to how `gh` works, too).